src/hle/rt64_application_window.cpp

```diff
@@ -13,7 +13,9 @@
 #   include <ShellScalingAPI.h>
 #elif defined(__linux__)
 #   define Status int
-#   include <X11/extensions/Xrandr.h>
+#   if !defined(RT64_SDL_WINDOW_VULKAN)
+#      include <X11/extensions/Xrandr.h>
+#   endif
```

Comment:
There's more X11-specific code under the `__linux__` macro that needs to be handled in this file.